safep/TI.py

```diff
@@ -49,8 +65,53 @@ def plot_TI(cumulative, perWindow, width=8, height=4, PDFtype='KDE', hystLim=(-1
     
     return fig, [cumAx,eachAx] 
 
-def make_harmonicWall(FC=10, targetFC=0, targetFE=1, upperWalls=1, schedule=None, numSteps=1000, targetEQ=500, name='HW', lowerWalls=None):
-    HW = {'name':name, 'targetFC':targetFC, 'targetFE':targetFE, 'FC':FC, 'upperWalls':upperWalls, 'schedule':schedule, 'numSteps':numSteps, 'targetEQ':targetEQ, 'lowerWalls':lowerWalls}
+def make_harmonicWall(FC=10, targetFC=0, targetFE=1, upperWalls=1, schedule=None, numSteps=1000, targetEQ=500, name='HW', lowerWalls=None, lambdaExp=1., decoupling=None):
+    '''
+    This method should rarely be used, as long as a log file exists that can be parsed by parse_Colvars_log
+    and the result fed to make_harmonicWall_from_Colvars

```

Comment:
Agreed. I actually want to replace this dict with a dataclass or namedTuple and do away with this function all together. Do you think we'll ever need to change the values of the attributes after creation?

Reply:
Right now I don't see a situation where the values would be changed, however it's not something that is completely unthinkable.

Would there be significant benefits to replacing the dict with another data type? Trying to judge if it's worth the extra work, since the dict seems to do the job.